_maps/map_files/tramstation/tramstation.dmm

```diff
@@ -20044,6 +20044,15 @@
 /obj/structure/cable,
 /turf/open/floor/catwalk_floor,
 /area/maintenance/tram/mid)
+"fEi" = (
+/obj/structure/industrial_lift/tram{
+	icon_state = "invisible"
```

Comment:
hm

Reply:
yea i noticed that the tram floors aren't actually subtypes, they were just var edited to change their icon. and invisible is actually a floor icon. it allows me to place a tram floor under the thrusters without having a floor there so as to pull the thrusters along without having an ugly floor under it. its pretty neat and could probably become an actual subtype for the lift floors perhaps